Description: Install Java modules in the correct path
 This patch corrects the installation of the native Java modules
 to go to the path given by the Debian Java Policy. This helps
 to later use the simple install file to get them to the right
 location in the package. 
Author: Gert Wollny <gw.fossdev@gmail.com
Last-Update: 2016-03-26

Index: VTK-6.3.0/CMake/vtkJavaWrapping.cmake
===================================================================
--- VTK-6.3.0.orig/CMake/vtkJavaWrapping.cmake
+++ VTK-6.3.0/CMake/vtkJavaWrapping.cmake
@@ -79,7 +79,7 @@ function(vtk_add_java_wrapping module_na
     install(TARGETS ${module_name}Java
       EXPORT ${VTK_INSTALL_EXPORT_NAME}
       RUNTIME DESTINATION ${VTK_INSTALL_RUNTIME_DIR} COMPONENT RuntimeLibraries
-      LIBRARY DESTINATION ${VTK_INSTALL_LIBRARY_DIR} COMPONENT RuntimeLibraries
+      LIBRARY DESTINATION ${VTK_INSTALL_LIBRARY_DIR}/jni COMPONENT RuntimeLibraries
       ARCHIVE DESTINATION ${VTK_INSTALL_ARCHIVE_DIR} COMPONENT Development)
   endif()
 endfunction()
